src/Tizen.NUI/src/internal/Common/PropertyHelper.cs

```diff
@@ -209,6 +209,17 @@ private static object ObjectIntToFloat(object value)
             return value;
         }
 
+        private static object ObjectVector4ToFloat(object value)
+        {
+            Type type = value.GetType();
+            if (type.Equals(typeof(Vector4)))
+            {
+                return ((Vector4)value).X;
+            }
+
+            return value;
```

Comment:
It is type casting of normal C# programmer, I wonder what is your intent?
``` c#
 if (value is Vector4 vector) 
 {
    return vector.X;
 }
 return value;
```

Reply:
I took your opinion in my next patch https://github.com/Samsung/TizenFX/pull/2866.